### DISCUSSION TITLE
allow modifiying admin-ui next.config.js

### DISCUSSION THREAD

**Discussion by shreyas-jadhav** [keystonejs/keystone#8855] [Feature requests]
i want to modify headers() from next.config.js so that i can allow access to next-auth api routes from my front end app

**Comment by dcousens** [comment]
@shreyas-jadhav can you elaborate on exactly what you're trying to do?
Maybe with an example?

**Comment by Nmargolis** [comment]
The addition of the ability to configure the `basePath` was great! It would also be great to be able to modify the `trailingSlash` Next config option. For our use-case, we need to be able to set it to `true`. We're using an external system to route traffic to our Keystone App, and that system automatically adds trailing slashes. Without the ability to set `trailingSlash` to true, this results in endless redirects. Our only solution currently would be to downgrade to a much older version of Keystone so we could use the [getAdditionalFiles workaround](https://github.com/borisno2/keystone-in-next-auth-demo/blob/main/keystone.ts#L19). When we downgrade to a version below 5.3.0, we then run into the [issue of hanging builds](https://github.com/keystonejs/keystone/issues/7071).
